package.json

```diff
@@ -18,6 +18,8 @@
     "express": "^4.18.1",
     "fast-kde": "^0.2.1",
     "format": "^0.2.2",
+    "lodash.debounce": "^4.0.8",
+    "moment": "^2.30.1",
```

Comment:
Probably added by mistake?

Reply:
I saw it being used to check if the credentials are expired. But probably not necessary at this point, could be installed when we add the logic for it.

Reply:
It would be good to avoid "moment" because it's a quite big dependency, I think we can do similar checks by using simple timestamps.
Moment used to be useful to write dates and time in a locale friendly manner, but nowadays we have a lot of thing natively in browsers API (for example https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString).

"debounce" could be useful in some cases but probably not for this.